docs/provisioner/provisioning-api.md

```diff
@@ -87,6 +87,30 @@ The mutation returns OperationID allowing to retrieve the operation status.
 
 The mutation returns OperationID allowing to retrieve the operation status.
 
+### Upgrade Shoot mutation
+
+***upgradeShoot*** mutation upgrades Gardener Shoot cluster configuration, triggering shoot reconciliation.
```

Comment:
Shoot or shoot?

Reply:
Good catch! Gardener capitalizes it, so I think it's `Shoot`.